src/components/Box/BoxConstants.js

```diff
@@ -0,0 +1,352 @@
+import styles from "./Box.module.scss";
+
+export const OPACITY = Object.freeze({
+    DISABLED: styles.opacityDisabled,
+})
+
+export const OVERFLOW = Object.freeze({
+    AUTO: styles.overflowAuto,
+    HIDDEN: styles.overflowHidden,
+    VISIBLE: styles.overflowVisible,
+    SCROLL: styles.overflowScroll,
+})
+
+export const OVERFLOW_X = Object.freeze({
+    AUTO: styles.overflowxAuto,
+    HIDDEN: styles.overflowxHidden,
+    VISIBLE: styles.overflowxVisible,
+    SCROLL: styles.overflowxScroll,
+})
+
+export const OVERFLOW_Y = Object.freeze({
+    AUTO: styles.overflowYauto,
+    HIDDEN: styles.overflowYhidden,
+    VISIBLE: styles.overflowYvisible,
+    SCROLL: styles.overflowYscroll,
+})
+
+export const BORDER = Object.freeze({
+    NONE: styles.border0, DEFAULT: styles.border,
+})
+
+export const BORDER_TOP = Object.freeze({
+    NONE: styles.borderTop0, DEFAULT: styles.borderTop,
+})
+
+export const BORDER_END = Object.freeze({
+    NONE: styles.borderEnd0, DEFAULT: styles.borderEnd,
+})
+
+export const BORDER_BOTTOM = Object.freeze({
+    NONE: styles.borderBottom0, DEFAULT: styles.borderBottom,
+})
+
+export const BORDER_START = Object.freeze({
+    NONE: styles.borderStart0, DEFAULT: styles.borderStart,
+})
+
+export const BORDER_COLOR = Object.freeze({
+    UI_BORDER_COLOR: styles.borderColorUiBorderColor, LAYOUT_BORDER_COLOR: styles.borderColorLayoutBorderColor,
+})
+
+export const ROUNDED = Object.freeze({
+    NONE: styles.rounded0, SMALL: styles.roundedSmall, MEDIUM: styles.roundedMedium, BIG: styles.roundedBig,
+})
+
+export const ROUNDED_TOP = Object.freeze({
+    NONE: styles.roundedTop0, SMALL: styles.roundedTopSmall, MEDIUM: styles.roundedTopMedium, BIG: styles.roundedTopBig,
+})
+
+export const ROUNDED_END = Object.freeze({
+    NONE: styles.roundedEnd0, SMALL: styles.roundedEndSmall, MEDIUM: styles.roundedEndMedium, BIG: styles.roundedEndBig,
+})
+
+export const ROUNDED_BOTTOM = Object.freeze({
+    NONE: styles.roundedBottom0,
+    SMALL: styles.roundedBottomSmall,
+    MEDIUM: styles.roundedBottomMedium,
+    BIG: styles.roundedBottomBig,
+})
+
+export const ROUNDED_START = Object.freeze({
+    NONE: styles.roundedStart0,
+    SMALL: styles.roundedStartSmall,
+    MEDIUM: styles.roundedStartMedium,
+    BIG: styles.roundedStartBig,
+})
+
+export const SHADOW = Object.freeze({
+    SMALL: styles.shadowSmall, MEDIUM: styles.shadowMedium, LARGE: styles.shadowLarge,
+})
+
+export const MARGIN = Object.freeze({
+    NONE: styles.m0,
+    AUTO: styles.mAuto,
+    XS: styles.mXs,
+    SMALL: styles.mSmall,
+    MEDIUM: styles.mMedium,
+    LARGE: styles.mLarge,
+    XL: styles.mXl,
+    XXL: styles.mXxl,
+    XXXL: styles.mXxxl,
+})
+
+export const MARGIN_X = Object.freeze({
+    NONE: styles.mx0,
```

Comment:
in my personal opinion classes name as mx0 are less readable then marginxo
but this is only my opinion

Reply:
do not agree. 
and anyway the style names are not visible to the api

Reply:
per now kept my structure.